### PR TITLE
Update set-up-your-team-hierarchy.md

### PR DESCRIPTION
**Pull request by Harold-Mitts**
2 Notes section at https://learn.microsoft.com/en-us/microsoftteams/set-up-your-team-hierarchy#apply-your-hierarchy referred to requirements for using Preview version of Teams PowerShell module. This capability is not in the release version, so this document is updated to reflect. 

additionally, the section "### Install the Teams PowerShell module" has reference to the preview module. Updated this as well and clarified some of the text.

### CODE REVIEW

File: Teams/set-up-your-team-hierarchy.md [hunk 1]
```diff
@@ -144,11 +144,8 @@ Los Angeles Store,West Regional Zone,204a1287-2efb-4a8a-88e0-56fbaf5a2389,Large,
 
 ## Apply your hierarchy
 
-> [!NOTE] 
-> To perform this step, you must install and use the Teams PowerShell public preview module from the PowerShell Gallery. For steps on how to install the module, see Install Teams PowerShell.
-
 > [!NOTE]
-> Government Community Cloud (GCC) customers must use [cmdlet preview version 2.4.0-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.4.0-preview) or later to ensure data is routed to the GCC environment, rather than the public cloud environment.
+> To perform this step, you'll use "Microsoft Teams" PowerShell cmdlets. You can use version 4.6.0 or later of the Microsoft Teams cmdlets. This also applies for Government Community Cloud (GCC) customers.
```
Comment (JohanFreelancer9) [inline Teams/set-up-your-team-hierarchy.md:148]:
```suggestion
> To perform this step, you must use Microsoft Teams PowerShell cmdlets. You should use version 4.6.0 or later of the Microsoft Teams cmdlets. This also applies to Government Community Cloud (GCC) customers.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Harold-Mitts, kindly commit the suggested change. Many thanks.

**Comment by tiburd** [comment]
@Harold-Mitts 
Hi,
This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). 

Also, at your earliest convenience, please complete your account setup so that your GitHub and Microsoft accounts are connected:
- [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main)

**Comment by tiburd** [comment]
@MikePlumleyMSFT 
Do you approve of these changes?
Thanks!

cc: @scanum

**Pull request merged**